packages/nc-gui/components/smartsheet/Grid.vue

```diff
@@ -412,8 +411,7 @@ reloadViewDataHook.trigger()
               </th>
             </tr>
           </thead>
-          <!-- this prevent select text from field if not in edit mode -->
-          <tbody @selectstart.prevent>
```

Comment:
should this be removed? I think it was something that was added for multi-select?

Reply:
Something wrong happened while rebasing, will revert these changes :+1: